ADAL/src/cache/ADTokenCacheKey.h

```diff
@@ -36,6 +36,8 @@
     NSString* _clientId;
 }
 
+
+
```

Comment:
Keep one empty line for code style consistency? :)

Reply:
Sure :)